bundles/org.openhab.binding.fenecon/README.md

```diff
@@ -0,0 +1,146 @@
+
+```java
+Thing fenecon:fenecon:1 "FENECON Home" [hostname="192.168.1.11", refreshInterval=5]
+```
+
+### demo.items
+
+```java
+// Sitemap Items
+Group   Home                    "MyHome"              <house>                                 ["Indoor"]
+Group   GF                      "GroundFloor"         <groundfloor>          (Home)           ["GroundFloor"]
+// Utility room
+Group   GF_UtilityRoom          "Utility room"        <energy>               (Home, GF)       ["Room"]
+Group   GF_UtilityRoomSolar     "Utility room solar"  <solarplant>           (GF_UtilityRoom) ["Inverter"]
+
+// FENECON items
+String               EssState               <text>         (GF_UtilityRoomSolar) ["Status"]                {channel="fenecon:fenecon:1:state"}
+DateTime             LastFeneconUpdate      <time>         (GF_UtilityRoomSolar) ["Status"]                {channel="fenecon:fenecon:1:lastUpdate"}
+Number:Dimensionless EssSoc                 <batterylevel> (GF_UtilityRoomSolar) ["Measurement"]           {unit="%", channel="fenecon:fenecon:1:essSoc"}
+Number:Power         ChargerPower           <energy>       (GF_UtilityRoomSolar) ["Measurement", "Power"]  {channel="fenecon:fenecon:1:chargerPower"}
+Number:Power         DischargerPower        <energy>       (GF_UtilityRoomSolar) ["Measurement", "Power"]  {channel="fenecon:fenecon:1:dischargerPower"}
+Switch               EmergencyPowerMode     <switch>       (GF_UtilityRoomSolar) ["Switch"]                {channel="fenecon:fenecon:1:emergencyPowerMode"}
+
+Number:Power         ProductionActivePower  <energy>       (GF_UtilityRoomSolar) ["Measurement", "Power"]  {channel="fenecon:fenecon:1:productionActivePower"}
+Number:Power         SellToGridPower        <energy>       (GF_UtilityRoomSolar) ["Measurement", "Power"]  {channel="fenecon:fenecon:1:sellToGridPower"}
+Number:Energy        TotalSellEnergy        <energy>       (GF_UtilityRoomSolar) ["Measurement", "Energy"] {channel="fenecon:fenecon:1:sellToGridEnergy"}
+
+Number:Power         ConsumptionActivePower <energy>       (GF_UtilityRoomSolar) ["Measurement", "Power"]  {channel="fenecon:fenecon:1:consumptionActivePower"}
+Number:Power         BuyFromGridPower       <energy>       (GF_UtilityRoomSolar) ["Measurement", "Power"]  {channel="fenecon:fenecon:1:buyFromGridPower"}
+Number:Energy        TotalBuyEnergy         <energy>       (GF_UtilityRoomSolar) ["Measurement", "Energy"] {channel="fenecon:fenecon:1:buyFromGridEnergy"}
+
+// Examples of items for calculating the energy purchased and sold. Look at the demo.rules section. 
+Number:Currency      SoldEnergy "Total sold energy [%.2f €]"           <price> (GF_UtilityRoomSolar)
+Number:Currency      PurchasedEnergy "Total purchased energy [%.2f €]" <price> (GF_UtilityRoomSolar)
+
+
+```
+
+### demo.sitemap
+
+```perl
+sitemap demo label="FENECON Example Sitemap" {
+    Frame label="Groundfloor" icon="groundfloor" {
+        Group item=GF_UtilityRoom
+    }
+}
+```
+
+### demo.rules
+
+```java
```

Comment:
You can consider to create a tabstrip with `:::` even if it has one tab, it can show DSL, other tabs might be added later for JS, Python or other.

Reply:
Do you have an example with the tabstrip, unfortunately I can't get it to work?

Reply:
Check the pushover readme here https://github.com/openhab/openhab-addons/blob/8c29b5973e5fa1159f11b325eb3cd6723d690176/bundles/org.openhab.binding.pushover/README.md?plain=1#L118

Reply:
Thank you, I needed that example 😊